docs/docs/features/shared-albums.md

```diff
@@ -10,11 +10,14 @@ Album sharing allows you to share assets with other users or with people from ar
 When sharing shared albums, whats shared is:
 
 - The selected assets.
-- Metadata of the assets(Can be removed by sharing via link).
+- Metadata of the assets (Can be removed by sharing via link).
 
 ## Shared Album Features
 
 - Download all assets as zip file (Web only).
+  :::info Archive size limited.
```

Comment:
Maybe it will change with https://github.com/immich-app/immich/discussions/8446#discussioncomment-9123479